tests/pos-special/typeclass-encoding3.scala

```diff
@@ -619,8 +630,8 @@ object semiGroups {
 
   implicit def NatOps: Nat.type = Nat
 
-  def sum[T](xs: List[T])(implicit $ev: Monoid.Impl[T]) =
-    (Monoid.impl[T].unit /: xs)((x, y) => x `add` y)
+  def sum[T](xs: List[T])(implicit $ev: Monoid.common[T]) =
+    (Monoid.common[T].unit /: xs)((x, y) => x `add` y)
```

Comment:
I wonder if it could (aliased to) `apply`, so it becomes `Monoid[T].unit` which looks nicer and less bulky. Some of the alternative proposals had that short form.

Reply:
is this a typo?

`(implicit $ev: Monoid.common[T])` shouldn't that be `Monoid.Common[T]`? a type and not a term